lessons/intro/turtle/index.md

```diff
@@ -69,20 +74,41 @@ Ulož ho do adresáře pro dnešní lekci pod jménem `zelva.py`.
 
 > [note]
 > Jestli adresář pro dnešní lekci ještě nemáš, vytvoř si ho!
-> Pojmenuj ho třeba `03`.
+> Pojmenuj ho třeba `02`.
+
+> [warning]
+> Soubor nepojmenovávej `turtle.py` – z modulu `turtle` budeš importovat.
```

Comment:
This information is repeated in the next row, it would be better to delete one occurrence.

Reply:
It's repeated intentionally. People tend to not remember the first warning; some get stuck here if it's not repeated.

Reply:
Then I'm fine with that. :)